### PR TITLE
[fix] replace xeokit toolbar icons with octicons

### PR DESCRIPTION
**Pull request by Kharonus**
- bump octicon lib
- removed css overwrites
- added method to handle html rewrite for octicon insertion

### CODE REVIEW

File: frontend/src/app/features/bim/ifc_models/ifc-viewer/ifc-viewer.component.ts [hunk 1]
```diff
@@ -130,13 +132,35 @@ export class IFCViewerComponent implements OnInit, OnDestroy, AfterViewInit {
           this.ifcData.projects,
         );
       });
+
+    this.toolbarIconSubscription = this.replaceXeokitToolbarIcons();
+  }
+
+  private replaceXeokitToolbarIcons():Subscription {
```
Comment (HDinger) [inline frontend/src/app/features/bim/ifc_models/ifc-viewer/ifc-viewer.component.ts]: Please leave a comment here, explaining why all of that is necessary

**Review by HDinger — CHANGES_REQUESTED** [review]
I don't know why exactly, but for me all icons are rendered twice

Bildschirmfoto 2024-04-18 um 09 04 54

**Comment by Kharonus** [comment]
Is this behaviour somehow reproducable? I'll add some guard to it, nevertheless.

File: frontend/src/app/features/bim/ifc_models/ifc-viewer/ifc-viewer.component.ts (hunk 1 above)
Comment (Kharonus) [reply frontend/src/app/features/bim/ifc_models/ifc-viewer/ifc-viewer.component.ts]: done

**Review by HDinger — APPROVED** [review]

**Pull request merged**